spring-cloud-generator/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>spring-cloud-gcp</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>4.1.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+    </parent>
+
+    <groupId>com.google.api</groupId>
+    <artifactId>spring-cloud-generator</artifactId>
+    <name>Spring Code Generator for Google Cloud</name>
+    <description>Spring Code Generator for Google Cloud</description>
+
+    <properties>
+        <!-- [gapic-test]: for compiling test protos -->
+        <protobuf.version>3.21.12</protobuf.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.api</groupId>
+            <artifactId>gapic-generator-java</artifactId>
+            <version>2.17.0</version>
```

Comment:
@blakeli0 Can we add `gapic-generator-java` to the [gapic-generator-java-bom](https://github.com/googleapis/gapic-generator-java/tree/main/gapic-generator-java-bom) so that we don't duplicate version specs?

Reply:
I think we could, not sure why it was excluded initially though, maybe @suztomo knows? We might be able to extend the protobuf version from [here](https://github.com/googleapis/gapic-generator-java/blob/eee46fe2cbc466549115b943a79d7863bdc74c40/gapic-generator-java-pom-parent/pom.xml#L33) as well.

Reply:
No concern in adding it to the BOM. Because it's not a user-facing artifact, we didn't add it to the BOM so far.

Reply:
Thanks! Added to BOM in https://github.com/googleapis/gapic-generator-java/pull/1645; will update this PR when this becomes available in the next release.

Reply:
Added `dependencyManagement` block with the newest release of gapic-generator-java-bom  2.18.0.